### PR TITLE
Add the User Name and Category from Backend to Map and AR

### PR DESCRIPTION
**Pull request by artur-varosyan**
This request brings the following:

Additions:
* _Category_ class now features a method _getCategoryById()_ with updated category IDs from the backend
* _ARActivity_ now shows the contributor name again
* _MapsAcitivity_ now shows the contributor name again and filtering is enabled again

Bugs fixed:
* Fix bug where the filter would automatically reset the maximum distance to the lower bound in Map

All suggestions are welcome!

### CODE REVIEW

**Review by Lingtao-Cai — APPROVED** [review]
👍

**Review by Manoutofpower — APPROVED** [review]

**Review by HellinSun — APPROVED** [review]
It's a great idea to put a map in Category class! Great Job!

**Review by arfi2212 — APPROVED** [review]
I really like the getCategoryById addition! It will help make my code a lot cleaner too, so thanks Artur! <3

File: app/src/main/java/com/example/ar_reshare/MapsActivity.java [hunk 1]
```diff
@@ -258,6 +258,7 @@ public void onClick(View v) {
                     if (!chip.isChecked()) {
                         chip.setChecked(false);
                         chip.setChipBackgroundColor(ColorStateList.valueOf(UNCHECKED_CHIP_COLOUR));
+                        chip.setChipBackgroundColor(ColorStateList.valueOf(UNCHECKED_CHIP_COLOUR));
```
[the remark this answers is not in the document]
Comment (artur-varosyan) [reply app/src/main/java/com/example/ar_reshare/MapsActivity.java]: Thanks, I missed that! Now it is changed :)

**Comment by artur-varosyan** [comment]
Thanks, everyone!

**Pull request merged**